atomix/cluster/src/main/java/io/atomix/raft/metrics/LeaderMetrics.java

```diff
@@ -16,23 +16,87 @@
  */
 package io.atomix.raft.metrics;
 
+import io.prometheus.client.Counter;
+import io.prometheus.client.Gauge;
 import io.prometheus.client.Histogram;
 
 public class LeaderMetrics extends RaftMetrics {
+  private static final String FOLLOWER_LABEL = "follower";
 
   private static final Histogram APPEND_LATENCY =
       Histogram.build()
-          .namespace("atomix")
+          .namespace(NAMESPACE)
           .name("append_entries_latency")
           .help("Latency to append an entry to a follower")
-          .labelNames("follower", "partitionGroupName", "partition")
+          .labelNames(FOLLOWER_LABEL, PARTITION_GROUP_NAME_LABEL, PARTITION_LABEL)
           .register();
 
+  private static final Counter APPEND_RATE =
+      Counter.build()
+          .namespace(NAMESPACE)
+          .name("append_entries_rate")
+          .help("The rate of entries appended (counting entries, not their size)")
```

Comment:
:wrench: The value is not really the rate. It is the total count. When we visualize it, we use the `rate()` function. So it might be more accurate to specify it as the total count in the help description.